client/src/pages/register/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function Register(){
+    return(
+        <div>
+            Home
```

Comment:
I assume you'll be adding more anyway, but noticed you're sending back "Home" for the Register. Won't hold up the review, just wanted to point it out.